scripts/shellcheck_job.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+import argparse
+import pathlib
+import subprocess
+import sys
+import xmltodict
+
+
+def parse_args(args: list[str]) -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument(
+        "job_xml",
+        nargs="*",
+        help="The job XML file(s) to process",
+    )
+    return parser.parse_args(args)
+
```

Comment:
This spacing isn't pep8 but I'll let it slide if that's what you prefer ;-)

Reply:
good catch, I ran `black` and repushed.